### ISSUE TITLE
Create enforcer v1.399.0

### ISSUE THREAD

**Issue by satishviswanathan** [casdoor/casdoor#2270]
I'm seeing a blank screen when trying to create enforcer

![image](https://github.com/casdoor/casdoor/assets/17437409/e94e1499-5f66-45c6-8e30-1defcec946d8)

**Comment by hsluoyz** [comment]
@satishviswanathan use latest version

**Comment by farhaven** [comment]
I kinda bisected this to "somewhere" between 1.392.0 and 1.393.0. To reproduce:

1. Use this docker-compose file to start a MariaDB server and casdoor:
```yaml
version: "3.9"
services:
  mariadb:
    image: "mariadb:latest"
    ports:
      - "3306:3306"
    environment:
      - "MARIADB_ROOT_PASSWORD=secret"
      - "MARIADB_DATABASE=casdoor"
      - "MARIADB_USER=casdoor"
      - "MARIADB_PASSWORD=casdoor"
  casdoor:
    image: "casbin/casdoor:v1.393.0" # Or anything after v1.393.0
    ports:
      - "8000:8000"
    restart: "on-failure"
    environment:
      - "dataSourceName=casdoor:casdoor@tcp(mariadb:3306)/"
```
2. Once Casdoor is reachable on http://localhost:8000, log in with the admin/123 creds, click "Authorization -> Enforcers".
3. In the enforcers overview, click "Add", but don't save yet. The UI looks normal.
4. Reload the page. The UI will be in a broken state as shown in @satishviswanathan 's screenshot
5. The enforcer can not be edited and the UI is in a broken state when clicking 'Add' in the overview.

**Comment by hsluoyz** [comment]
@satishviswanathan @farhaven fixed by: https://github.com/casdoor/casdoor/commit/6715efd7815a435a1815eb0e462ffe632c607607

**Comment by satishviswanathan** [comment]
THank you @hsluoyz . is the fix part of [v1.406.2](https://github.com/casdoor/casdoor/releases/tag/v1.406.2)
